src/js/modal/exercise-modal.js

```diff
@@ -128,11 +131,11 @@ const markupGiveRatingBtn = () =>
   });
 
 const markupButton = ({ text, iconId, className = '' }) => `
-  <button type="button" class="js-favorites button ${className}">
+  <button id="js-toggle-favorit" type="button" class="js-favorites button ${className}">
```

Comment:
this is a 'factory' function and could be used for making different buttons, therefore such 'id' is not correct (my fault with js-favorites class as well)